### PR TITLE
Checklist app reload acl

### PR DESCRIPTION
**Pull request by ArktinenSieni**

### CODE REVIEW

File: checklist-app/src/Futurice/App/Checklist/Types/Ctx.hs [hunk 1]
```diff
@@ -35,7 +36,8 @@ data Ctx = Ctx
     , ctxPostgres    :: Pool Postgres.Connection
     , ctxPRNGs       :: Pool (TVar CryptoGen)
     , ctxMockUser    :: !(Maybe FUM.UserName)
-    , ctxACL         :: Map FUM.UserName TaskRole
+    , ctxACL         :: TVar (Map FUM.UserName TaskRole)
+    , ctxManager     :: Manager
```
Comment (ArktinenSieni) [inline checklist-app/src/Futurice/App/Checklist/Types/Ctx.hs]: Not sure if necessary, even if [documentation](http://docs.futurice.com/haskell-mega-repo/futurice-prelude-1/Futurice-Prelude-Internal.html) did recommend to use `newManager`-function with care.

Comment (phadej) [inline checklist-app/src/Futurice/App/Checklist/Types/Ctx.hs]: Having (only) single manager in the app is good. :+1:

File: checklist-app/src/Futurice/App/Checklist/Types/Ctx.hs [hunk 2]
```diff
@@ -48,29 +50,17 @@ newCtx
...
+newCtx logger ci fumAuthToken fumBaseUrl (itGroupName, hrGroupName, supervisorGroupName) mockUser w = do
+    mgr <- newManager tlsManagerSettings
+    Ctx logger
+        <$> newTVarIO w
+        <*> pure w
+        <*> pure Dict
+        <*> createPool (Postgres.connect ci) Postgres.close 1 60 5
+        <*> createPool (mkCryptoGen >>= newTVarIO) (\_ -> return()) 1 3600 5
+        <*> pure mockUser
+        <*> ctxFetchGroups newTVarIO mgr logger fumAuthToken fumBaseUrl (itGroupName, hrGroupName, supervisorGroupName)
```
Comment (phadej) [inline checklist-app/src/Futurice/App/Checklist/Types/Ctx.hs]: I'd leave the initial ACL map empty, as it will be updated immediately by the job!

**Comment by phadej** [comment]
Also rebase on current `master`, I did some updates there.

File: checklist-app/src/Futurice/App/Checklist.hs [hunk 3]
```diff
@@ -352,12 +354,21 @@ makeCtx' Config {..} logger _cache = do
...
         cfgMockUser
         emptyWorld
     cmds <- withResource (ctxPostgres ctx) $ \conn ->
         Postgres.query_ conn "SELECT username, updated, cmddata FROM checklist2.commands ORDER BY cid;"
     let world0 = foldl' (\world (fumuser, now, cmd) -> applyCommand now fumuser cmd world) emptyWorld cmds
     atomically $ writeTVar (ctxWorld ctx) world0
-    pure (ctx, [])
+
+    let action = ctxFetchGroups
+            pure
```
Comment (phadej) [inline checklist-app/src/Futurice/App/Checklist.hs]: that argument isn't needed anymore, is it?

**Pull request merged**